h/traversal/__init__.py

```diff
@@ -23,6 +24,7 @@
     "Root",
     "AnnotationRoot",
     "AuthClientRoot",
+    "BulkAPIRoot",
```

Comment:
:+1: We could consider removing the `__all__` from this file as I don't think it's adding anything. The `# -*- coding: utf-8 -*-` too. Might be worth separate PRs at some point to just remove all of both of those things from h

Reply:
I think they just add another thing to maintain. As an added bonus it also makes it more annoying to find the class which actually implements the code whenever you follow an import as it dumps you in a file that just imports it again.

This makes sense for a public API, but I don't think it buys much internally.